diesel/src/pg/transaction.rs

```diff
@@ -1,6 +1,9 @@
 #![allow(dead_code)]
 use crate::backend::Backend;
-use crate::connection::{AnsiTransactionManager, TransactionManager};
+use crate::connection::{
+    commit_error_processor::CommitErrorProcessor,
+    {AnsiTransactionManager, TransactionManager},
+};
```

Comment:
Prefer flat imports here
```suggestion
use crate::connection::commit_error_processor::CommitErrorProcessor;
use crate::connection::{AnsiTransactionManager, TransactionManager};
```